queries.sql

```diff
@@ -1,55 +1,116 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * FROM animals WHERE name LIKE '%mon';
-SELECT name FROM animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-12-31';
-SELECT name FROM animals WHERE (escape_attempts < 3) AND neutered;
-SELECT date_of_birth from animals WHERE name IN ('Agumon', 'Pikachu');
-SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
-SELECT * FROM animals WHERE neutered;
-SELECT * FROM animals WHERE name NOT IN ('Gabumon');
-SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+SELECT * from animals WHERE   name Like '%mon';
+SELECT * from animals WHERE   date_of_birth between '2016-01-01' and '2019-12-31';
+SELECT * from animals WHERE   neutered = true and escape_attempts < 3;
+SELECT * from animals WHERE   name in ('Agumon', 'Pikachu');
+SELECT  name, escape_attempts from animals WHERE   weight_kg > 10.5;
+SELECT * from animals WHERE   neutered = true;
+SELECT * from animals WHERE   name <> 'Gabumon';
+SELECT * from animals WHERE  weight_kg >= 10.4 and weight_kg  <= 17.3;
 
 BEGIN;
-UPDATE animals
-SET species = 'unspecified';
-SELECT * FROM animals;
+
+   UPDATE animals
+   SET species = 'unspecified';
+
+   SELECT species from animals;
+
 ROLLBACK;
-SELECT * FROM animals;
+
+SELECT species from animals;
 
 BEGIN;
-UPDATE animals
-SET species = 'digimon'
-WHERE name LIKE '%mon';
-UPDATE animals
-SET species = 'pokemon'
-WHERE species IS NULL;
-SELECT * FROM animals;
+
+    UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+
+    UPDATE animals SET species = 'pokemon' WHERE species is null;
```

Comment:
- [ ] As requested in the project requirement, everything that populates tables with data should be saved in the `data.sql`. Therefore I suggest you move these codes into the `data.sql` file to follow the project requirement.

Reply:
@LionRouge1, thank you for a constructive feedback. Working on it.